lib/components/admin/trip-status.js

```diff
@@ -92,7 +90,7 @@ class TripStatus extends Component {
     return (
       <Full>
         <Header>
-          {this._renderStatusIcon()}
+          <FieldTripStatusIcon status={this._tripIsPlanned()} />
```

Comment:
The value passed to status is ambiguous here... are we passing a trip or a status? (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND#using_and)